### PR TITLE
Clarify the difference between _compact, revs_limit and _purge

### PR DESCRIPTION
**Pull request by jamiltz**
Relates to #483

### CODE REVIEW

File: swagger/paths/sg/admin.yaml [hunk 1]
```diff
@@ -269,7 +269,7 @@
...
    description: |
-     When an application deletes a document a tombstone revision is created, over time the number of tombstones can become significant. Tombstones allow document deletions to be propagated to other clients via replication. For some applications the replication of a tombstone may not be useful after a period of time. The purge command provides a way to remove the tombstones from a Sync Gateway database, recovering storage space and reducing the amount of data replicated to clients.
+     When an application deletes a document a tombstone revision is created, over time the number of tombstones can become significant. Tombstones allow document deletions to be propagated to other clients via replication. For some applications the replication of a tombstone may not be useful after a period of time. The purge command provides a way to remove the tombstones from a Sync Gateway database, recovering storage space and reducing the amount of data replicated to clients. Other clients are not notified when a revision has been purged; so in order to purge a revision from the system it must be done from all databases (on Couchbase Lite and Sync Gateway).
```
Comment (adamcfraser) [inline swagger/paths/sg/admin.yaml]: Although removing tombstones would be a common use case, purge doesn't actually have any relationship to the document's current revision state (active or tombstone).  The API docs should probably start with a more generic description of the functionality (it deletes the document in the bucket), and only mention the tombstone bit as one potential use case.

File: md-docs/ready/guides/sync-gateway/config-properties.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ Following are the configuration properties for a specific database:
...
 |`pool`|`string`|	Couchbase pool name. The default is the string `default`.|
 |`rev_cache_size`|`integer`|Size of the revision cache, specified as the total number of document revisions to cache in memory for all recently accessed documents. When the revision cache is full, Sync Gateway removes less recent document revisions to make room for new document revisions. Adjust this property to tune memory consumption by Sync Gateway, for example on servers with less memory and in cases when Sync Gateway creates many new documents and/or updates many documents relative to the number of read operations. The default is `5000`.|
-|`revs_limit`|`integer`|Maximum depth to which a document's revision tree can grow. The default is `1000`.|
+|`revs_limit`|`integer`|Maximum depth to which a document's revision tree can grow. This setting can be used to reduce the maximum size of Sync Gateway's metadata. Be aware that if the value is too low it may result in unexpected behaviour when conflicts occur. The default is `1000`. To purge individual revisions it is recommended to use the `/{db}/_purge` endpoint instead.|
```
Comment (adamcfraser) [inline md-docs/ready/guides/sync-gateway/config-properties.md]: It's a bit misleading to say that _purge can be used to purge individual revisions - it deletes the entire doc and rev tree.

Comment (jamiltz) [inline md-docs/ready/guides/sync-gateway/config-properties.md]: Good point. So instead, should we write "To purge individual **documents** [...]"?

**Comment by jamiltz** [comment]
@adamcfraser Updated per review:

    - Clarify that /_purge removes all revisions of a doc.
    - Mention TTL as well as a way to purge a single doc.

**Pull request merged**